vsts_ci/azure-pipelines.yml

```diff
@@ -23,6 +23,13 @@ jobs:
     steps:
       - template: policy/continuous-legal-status-policy-check.yml
 
+  - job: PublishStandaloneBinariesForWin32
```

Comment:
will this job be triggered during for PR?

Reply:
Just when it has tags